### ISSUE TITLE
enso-bundle tries to download missing engine

### ISSUE THREAD

**Issue by JaroslavTulach** [enso-org/enso#9753]
Daily [version from Apr 19, 2024](https://github.com/enso-org/enso/releases/download/2024.1.1-nightly.2024.4.19/)
demonstrates a problem. Download the right "enso-bundle-" first. If on Linux download [enso-bundle-2024.1.1-nightly.2024.4.19-linux-amd64.tar.gz](https://github.com/enso-org/enso/releases/download/2024.1.1-nightly.2024.4.19/enso-bundle-2024.1.1-nightly.2024.4.19-linux-amd64.tar.gz) and untar it. Then
```
$ git clone https://github.com/otulach/SCMrozdeleni
$ cd SCMrozdeleni
$ git checkout cc1f40174186b33a7b7ec43984bc49757f8be742
SCMrozdeleni$ ../../enso/bin/enso run src/Download.enso 
[WARN] [2024-04-19T15:35:30+02:00] [org.enso.launcher.cli.CLIRuntimeVersionManagementUserInterface] Engine 0.0.0-dev is missing.
Do you want to install the missing engine? [Y/n] 
```
**No**! I don't want to download anything. As written in #9231 - our current policy is to **never download anything**.

Expected behavior: don't ask to _install the missing engine_.

**Comment by JaroslavTulach** [comment]
If one [removes edition from package.yaml](https://github.com/otulach/SCMrozdeleni/commit/f182373a030fb4b02803d5e0dbc12b684fc54159#diff-e0ee4e21f8811c1171864cc68ea4005347b1b0ca70626026f251bf4111c2aa6eL7), the problem goes away. However that's not what we want our users to do. We just can't ask the `Do you want to install the missing engine? [Y/n]` question to begin with.

**Comment by JaroslavTulach** [comment]
- I believe this one got fixed by #10408
